_posts/2020-05-05-azure-ad-authentication-cypress.md

```diff
@@ -135,3 +126,5 @@ There you have it, we have managed to set up our Cypress tests to authenticate w
 Manual tests taking up all your time and not finding issues before they get to users? We can help!
 
 [Contact Mechanical Rock to Get Started!](https://www.mechanicalrock.io/lets-get-started)
+
```

Comment:
I think it might be worth putting this update message at the top of the article in italics.

Reply:
Roger